### PR TITLE
feat(Retroblur): add Retroblur theme

### PR DESCRIPTION
**Pull request by Motschen**
Retroblur is a spicetify theme that implements the retrowave/vaporwave aesthetic along with blur effects, a background and so much more:
- Retro-style color scheme
- Background picture
- Blur behind many objects
- Glow around highlighted hyperlinks
- Adapted Lyrics-Plus to show the previous and 3rd next line
- Fix some bugs in Lyrics-Plus
- Made the no lyrics found emoji bounce around the screen like the DVD screensaver
- Many small details and tweaks

Since it's based on the Ziro theme, following features also apply:
- long progress bar
- overhauled artist page
- rounded corners everywhere

### CODE REVIEW

**Comment by RaddedMC** [comment]
I like this! It'd be cool if the background could change image / colour based on the album art of the current song

File: Retroblur/color.ini [hunk 1]
```diff
@@ -0,0 +1,215 @@
+; COLOR EXPLANATION
```
Comment (morpheusthewhite) [inline Retroblur/color.ini:1]: There are several color schemes. You should provide a screenshot for each of them and structure the README as specified in the [contribution guidelines](https://github.com/morpheusthewhite/spicetify-themes/blob/master/CONTRIBUTING.md)

**Review by morpheusthewhite — CHANGES_REQUESTED** [review]

**Comment by morpheusthewhite** [comment]
Sorry for reviewing this late, I've been quite busy in the last month

**Comment by afonsojramos** [comment]
@Motschen do you think it still makes sense to merge your PR?

**Comment by kyrie25** [comment]
@Motschen I've noticed that the theme is currently still being maintained at your repository. Was wondering if you still wish to move it here or continue the development there?

**Comment by Motschen** [comment]
Yeah, I'm going to update this PR when I have some more time.

**Comment by afonsojramos** [comment]
@Motschen worry not, after #766 it will be easier to do it, and you'll be able to keep developing under your repo! I'll close this PR as adding to the repo this way is no longer the objective ✌️

**Pull request closed without merging**